docs/admin/baremetal-installation-guide.md

```diff
@@ -240,3 +244,28 @@ If you run into an issue with migration `V0611`, ensure that the user running Fl
 
 If you encounter other errors migrating the database, make sure that you are using the recommended version of Flyway
 (see [Prerequisites](#prerequisites)).
+
+# Server Deployment Troubleshooting
+
+* Whenever unexpected behaviour arises, be sure to check the build logs, located at `${CATALINA_BASE}/logs`. Read through the output of `catalina.out`, `catalina.<date>.log`, and `localhost.<date>.log`.
+* Ensure you are using the correct version of Java with Tomcat. This can be checked in `catalina.out`. In case you are not, append/modify the value of `$JAVA_HOME` in `setenv.sh` or `/etc/default/tomcat9`. 
+* If file access for `/storage/miso` is causing an error, [this](https://stackoverflow.com/questions/56827735/how-to-allow-tomcat-war-app-to-write-in-folder) may help.
+
+
+# Development Alternatives
+
+ If using the correct MySQL version for your database server is causing trouble, consider downloading [Docker](https://docs.docker.com/get-docker/) to use a Docker container. Creating the container (note: things prefixed with `VAR_` are a variable, and their value will be specified below):
```

Comment:
> If using the correct MySQL version for your database server is causing trouble

This wording sounds like you have the correct version installed and are having some problem with it. This is an alternative to installing the correct MySQL version, whether you're unable or just not wanting to install it.

> things prefixed with `VAR_` are a variable

The convention used in the rest of this document is to represent variables like `$CONTAINER_NAME`